2020/13xxx/CVE-2020-13670.json

```diff
@@ -1,18 +1,74 @@
 {
```

Comment:
{} is not allowed for {'reference_data': [{'name': 'https://www.drupal.org/sa-core-2020-011', 'refsource': 'CONFIRM', 'url': 'https://www.drupal.org/sa-core-2020-011'}]}